modules/ci_environment/manifests/transition_logs/remote_users.pp

```diff
@@ -1,45 +0,0 @@
-# class to generate remote log upload users.
-define ci_environment::transition_logs::remote_users (
-  $comment,
-  $ssh_key,
-  $home_dir
-){
-
-  account{$name:
-    comment  => $comment,
-    ssh_key  => $ssh_key,
-    home_dir => $home_dir,
-    shell    => '/usr/bin/rssh',
-    require  => Ext4mount['/srv/logs/log-1'],
-  }
-
-  file {"${home_dir}/cache":
-    ensure  => directory,
-    owner   => 'logs_processor',
-    group   => 'gds',
-    mode    => '0775',
-    require => Account[$name],
-  }
-
-  # Generate random cron times seeded with fqdn and username
-  $hour = fqdn_rand(8, $name)
-  $min  = fqdn_rand(59, $name)
-
-  # get backup target
-  # set a default of scp to localhost as the vagrant user.
-  # if testing stuff you will need to generate ssh keys
-  # and then add the public key to vagrant's authorized_keys.
-  $backup_target = hiera('ci_environment::transition_logs::backup_target', false)
```

Comment:
Deleting these lines negates the need for https://github.gds/gds/ci-deployment/pull/58.